packages/fluentui/docs/src/examples/components/Datepicker/Usage/DatepickerFormatExample.shorthand.tsx

```diff
@@ -0,0 +1,13 @@
+import { Datepicker } from '@fluentui/react-northstar';
+import * as React from 'react';
+
+const DatepickerFormatExample = () => {
+  const customFormat = (date: Date) => {
+    const day = date.getDate();
+    const year = date.getFullYear();
+    return `${day} of ${year}`;
+  };
+  return <Datepicker formatMonthDayYear={customFormat} today={new Date(2020, 8, 12, 0, 0, 0, 0)} />;
```

Comment:
Just curiosity `formatMonthDayYear` works with string patterns also? e.g `YY/MM/DD`

Reply:
Well, that would be up to the consumer, like they can have `customFormat` method defined like that:

```  const customFormat = (date: Date) => {
    const options = {
      weekday: 'short',
      year: 'numeric',
      month: 'short',
      day: '2-digit',
    };

    return date.toLocaleDateString('en-US', options);
  };
```